tests/test_other.py

```diff
@@ -7238,9 +7243,15 @@ def test_sixtyfour_bit_return_value(self):
     ''')
 
     # Run the test and confirm the output is as expected.
-    out = run_js('testrun.js', full_output=True)
-    self.assertContained('low = 5678', out)
-    self.assertContained('high = 1234', out)
+    out = run_js('testrun.js', full_output=True, engine=NODE_JS + ['--experimental-wasm-bigint'])
```

Comment:
What does `full_output` do?  Is it needed here?

Reply:
it allows getting stderr too i think, but you're right, it's not needed here...